docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -1026,201 +1026,134 @@ The following error message codes may appear on the app-server log. Use the foll
 
   **Action:**
 
-  No action required. 
-
-
-
-### ZWED0289I
-
-  JarMgr with id=%s invoked to startup with config=%s
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
+  No action required.
 
 
 
 ### ZWED0290I
 
-  Plugin (%s) loaded. Version: %s. Successful: %s% (%s/%s) Attempted: %s% (%s/%s)
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
-### ZWED0291I
-
-  Server is ready at %s Plugins successfully loaded: %s% (%s/%s)
+  Plugin *event.data.identifier* loaded. Version: *event.data.pluginVersion*. Successful: _(pluginsLoaded/event.count)*100)_ (*pluginsLoaded*/*event.count*) Attempted: _(pluginCount/event.count)*100)_ (*pluginsCount*/*event.count*)
 
   **Reason:**
 
-  TODO
+  Plugin event data identifier loaded successfully. Version is event data pluginVersion.
+  Successfully plugin loaded and event count, and attempted pluginCount and event count.
 
   **Action:**
 
-  TODO
+  No action Required.
 
 
 
 ### ZWED0292I
 
-  Plugin (%s) loaded. Version: %s.
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
-### ZWED0293I
-
-  Handling scan plugin request from worker=%d
+  Plugin *identifier* loaded. Version: *plugin* version.
 
   **Reason:**
 
-  TODO
+  Plugin identifier loaded successfully and the plugin version for the same is *plugin version*.
 
   **Action:**
 
-  TODO
+  No action Required.
 
 
 
 ### ZWED0294I
 
-  Successfully loaded %d recognizers for '%s' into config
+  Successfully loaded *recognizers.length* recognizers for *appId* into config
 
   **Reason:**
 
-  TODO
+  Successfully loaded recognizers length for plugin recognizers into config at path *workspace/app-server/ZLUX/pluginStorage/org.zowe.zlux.ng2desktop/recognizers*.
```

Comment:
I am not confident about this path, I remember recognizers being stored at the plugin level, not all on the Desktop so just double check this or remove it. And also include in this sentence the exact variable name used so it's easier to read